src/classes/User/User.ts

```diff
@@ -63,12 +64,14 @@ export class User extends PassageBase {
 
             const users = response.users;
             if (!users.length) {
-                throw new PassageError('User not found.');
+                throw new ResponseError(
```

Comment:
Still want to throw a PassageError to emulate a network error here since this method should essentially have the same behavior as the `get`